utility/nsfs_server_utils.py

```diff
@@ -133,6 +133,25 @@ def create_tls_key_and_cert(credentials_dir):
     return f"{credentials_dir}/tls.crt"
 
 
+def get_system_json(config_root=config.ENV_DATA["config_root"]):
+    """
+    Get the content of the NSFS system.json file
+
+    Args:
+        config_root(str): The full path to the configuration root directory on the remote machine
+
+    Returns:
+        dict: The content of the system.json file
+    """
+    conn = SSHConnectionManager().connection
+    retcode, stdout, _ = conn.exec_cmd(f"sudo cat {config_root}/system.json")
+    if retcode != 0:
+        raise MissingFileOrDirectory(
+            f"system.json file not found in {config_root}: {stdout}"
```

Comment:
I think we need to log `stderr` here

Reply:
Good point - more details here would make it easier to see that the recent failures have been due to permission issues and not the absence of the file.

Added in https://github.com/red-hat-storage/noobaa-sa-ci/pull/29/commits/7460465412f54558e66857a733de49bc3ae02c7d